Mathlib/Data/Fin/Tuple/Sort.lean

```diff
@@ -107,6 +109,83 @@ theorem monotone_sort (f : Fin n → α) : Monotone (f ∘ sort f) := by
   exact (monotone_proj f).comp (graphEquiv₂ f).monotone
 #align tuple.monotone_sort Tuple.monotone_sort
 
+/-- All the elements `· ≤ a` appear the start of a sorted tuple -/
```

Comment:
This docstring is wrong; the theorem says "all the elements at the start are `≤ a`"; this docstring belongs on `sort_lt_at_start_of_monotone'` instead.

Reply:
This was the doc string from your example on the Zulipchat. I changed it to another description